src/dbos-executor.ts

```diff
@@ -116,49 +114,38 @@ export const DBOS_QUEUE_MAX_PRIORITY = 2 ** 31 - 1; // 2,147,483,647
 
 /* Interface for DBOS configuration */
 export interface DBOSConfig {
-  // Public fields
   name?: string;
-  readonly databaseUrl?: string;
-  readonly userDbclient?: UserDatabaseName;
-  readonly userDbPoolSize?: number;
-  readonly sysDbName?: string;
-  readonly sysDbPoolSize?: number;
-  readonly logLevel?: string;
-  readonly otlpTracesEndpoints?: string[];
-  readonly otlpLogsEndpoints?: string[];
-  readonly adminPort?: number;
-  readonly runAdminServer?: boolean;
-
-  // Internal fields
-  poolConfig?: PoolConfig;
-  readonly telemetry?: TelemetryConfig;
-  readonly system_database?: string;
-  readonly env?: Record<string, string>;
-  readonly application?: object;
-  readonly http?: {
-    readonly cors_middleware?: boolean;
-    readonly credentials?: boolean;
-    readonly allowed_origins?: string[];
-  };
+  databaseUrl?: string;
+  userDbclient?: UserDatabaseName;
+  userDbPoolSize?: number;
+  sysDbName?: string;
+  sysDbPoolSize?: number;
+  logLevel?: string;
+  addContextMetadata?: boolean;
```

Comment:
Doable in another PR: this is used to inject contextual data to the logger _console_ output, which imho is unnecessary. We should get ride of it entirely.

Reply:
Should we also get rid of `dboslogger.test.ts` which exercises this?

Reply:
Yeah, let's get rid of this